### PR TITLE
Shrijana makeapicall

### PR DESCRIPTION
**Pull request by ShrijanaCodes**
@timslaglezoom @michaelZoomDA I have added a feature to submit email via a form and get the api response in the existing codebase for this repo. I have also modified the README file. Could you please review my code and accept/give feedback? Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,9 @@ Clone the repo using git clone.
 > Start the node app.
```
Comment (timslaglezoom) [inline README.md:38]: @ShrijanaCodes, let's make sure to add clear instructions to the readme. Step one clone.... step 2... run node index.js.... etc.. and what does the app do?

**Review by michaelpurnellZoom — COMMENTED** [review]
Looks good go ahead and merge

File: README.md (hunk 1 above)
Comment (ShrijanaCodes) [reply README.md:38]: Sounds good, Tim! On it.

Comment (ShrijanaCodes) [reply README.md:38]: @timslaglezoom I have updated the README, Tim.

**Pull request merged**